apps/judicial-system/backend/src/app/modules/event/event.service.ts

```diff
@@ -32,16 +33,18 @@ const errorEmojis = [
 ]
 
 const caseEvent = {
-  CREATE: ':new: Krafa stofnuð',
-  CREATE_XRD: ':new: Krafa stofnuð í gegnum Strauminn',
-  EXTEND: ':recycle: Krafa framlengd',
+  CREATE: ':new: Mál stofnað',
+  CREATE_XRD: ':new: Mál stofnað í gegnum Strauminn',
+  EXTEND: ':recycle: Mál framlengt',
   OPEN: ':unlock: Krafa opnuð fyrir dómstól',
-  SUBMIT: ':mailbox_with_mail: Krafa send til dómstóls',
-  RESUBMIT: ':mailbox_with_mail: Krafa send aftur til dómstóls',
-  RECEIVE: ':eyes: Krafa móttekin',
+  OPEN_INDICTMENT: ':unlock: Ákæra opnuð fyrir dómstól',
+  SUBMIT: ':mailbox_with_mail: Sent',
+  RESUBMIT: ':mailbox_with_mail: Sent aftur',
+  RECEIVE: ':eyes: Móttekið',
+  CLOSE_INDICTMENT: ':white_check_mark: Lokið',
   ACCEPT: ':white_check_mark: Krafa samþykkt',
   REJECT: ':negative_squared_cross_mark: Kröfu hafnað',
-  DELETE: ':fire: Krafa dregin til baka',
+  DELETE: ':fire: Afturkallað',
   SCHEDULE_COURT_DATE: ':timer_clock: Kröfu úthlutað fyrirtökutíma',
```

Comment:
This event can also happen for indictments. Prosecutors get COURT_DATE notifications for indictments and the event is triggered there.